cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -213,6 +219,55 @@ export function loadAnnotationsAsync(
     };
 }
 
+function importTask(): AnyAction {
+    const action = {
+        type: TasksActionTypes.IMPORT_TASK,
+        payload: {},
+        importing: true,
```

Comment:
literal values should not be a part of payload because we can specify them in reducer based on action type